src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -116,11 +116,12 @@ def __init__(
         request_throughputs: List[float] = [],
         request_latencies: List[int] = [],
         time_to_first_tokens: List[int] = [],
-        inter_token_latencies: List[List[int]] = [[]],
+        inter_token_latencies: List[int] = [],
         output_token_throughputs: List[float] = [],
         output_token_throughputs_per_request: List[int] = [],
         num_output_tokens: List[int] = [],
         num_input_tokens: List[int] = [],
+        old_inter_token_latencies: List[List[int]] = [[]],
```

Comment:
I dont love this name.
At some point in the future, we won't know what old means here.

Reply:
Yeah same. Do you have any suggestions?

Reply:
What is its purpose right now?
is one token latency and the other request?

Reply:
It's just the ITL we used to calculate before the new ITL. I'm keeping it to allow ITL vs token position plot.

Reply:
That makes sense to me.

If I am understanding this correctly, inter_token_latencies is the flatten version of the old_inter_token_latencies.
This new one is more a representation of chunked or response inter token latency.
So perhaps we do:
old_inter_token_latencies -> inter_token_latencies
inter_token_latencies -> chunked_inter_token_latencies

Reply:
> If I am understanding this correctly, inter_token_latencies is the flatten version of the old_inter_token_latencies.

Not really. I would say it's more accurate to say "merged" than flattened since the new one is technically a normalized average of ITLs. 

> So perhaps we do:
old_inter_token_latencies -> inter_token_latencies
inter_token_latencies -> chunked_inter_token_latencies

I think it's best we keep the name of the ITL we display as "inter_token_latencies". And actually the old ITL is more closer to chunked ITL since it weighs more heavily on how many tokens are contained in each response, so maybe we can call the old one as `chunked_inter_token_latencies`.

Reply:
So i did NOT understand correctly. LoL.
Thank you for your patience as I get on board.
I am okay with chunked_inter_token_latencies as a different name for the old_inter_token_latencies variable

Reply:
Changed the name.